spec/usecases/reactions/update_materials_spec.rb

```diff
@@ -73,23 +73,36 @@
 
   describe '#execute!' do
     context 'when sample exists' do
+      before do
+        ReactionsStartingMaterialSample.create!(
+          reaction: reaction, sample: sample, reference: true, equivalent: 1
+        )
+      end
+
+      it 'does not update reaction-SVG from sample model' do
+        # Capturing SVG::ReactionComposer:new is the most straightforward way I could think of to test updates to the reaction-SVG.
+        # Since the updates to the reaction-SVG are a side-effect they are difficult to test.
+        expect(SVG::ReactionComposer).to receive(:new).once # only one final update to reaction-SVG once reaction is saved
+        described_class.new(reaction, starting_materials, user).execute!
```

Comment:
All other it-blocks are built with a "given..when..then" structure (aka. "setup..call the method..expect something") except this one ("given..than..when"). It-Blocks are easier to refactor if they follow the same pattern.

```suggestion
        allow(SVG::ReactionComposer).to receive(:new) # this enables the have_received-matcher
        described_class.new(reaction, starting_materials, user).execute!
        expect(SVG::ReactionComposer).to have_received(:new)
```

Let's refactor this spec a little bit in a later step...